src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1,12 +1,4 @@
-UPDATE tab_calculation_method_rates SET formula='CP * f_int * V * kcat * (fu * max(C_pls;0))^alpha / (Km^alpha + (fu * max(C_pls;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='BrainActiveInfluxFromPlasma_Hill';
-UPDATE tab_calculation_method_rates SET formula='Min(CP * f_int * V * kcat * (fu * max(C_pls;0))^alpha / ( Km^alpha + (fu * max(C_pls;0))^alpha) ; P_endothelial * SA * fu * C_pls) + CP * f_int * V * kcat * (K_water_int * max(C_int;0))^alpha / (Km^alpha + (K_water_int * max(C_int;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='BrainPgPToPlasma_Hill';
-UPDATE tab_calculation_method_rates SET formula='CP * f_cell * V * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha + (K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='KidneyActiveEffluxToUrine_Hill';
-UPDATE tab_calculation_method_rates SET formula='(1-EHC_ContinuousFraction) * CP * f_cell * V * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha + (K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='LiverActiveEffluxToBile_Hill';
-UPDATE tab_calculation_method_rates SET formula='EHC_ContinuousFraction * CP * f_cell * V * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha + (K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='LiverActiveEffluxToDuodenum_Hill';
-UPDATE tab_calculation_method_rates SET formula='CP * kcat * V * (K_water * max(C;0))^alpha / (Km^alpha + (K_water * max(C;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='MetabolizationSpecific_Hill';
-UPDATE tab_calculation_method_rates SET formula='TM * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha+(K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='MucosaActiveEffluxToLumen_Hill';
-UPDATE tab_calculation_method_rates SET formula='min(TM * kcat * max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha / (Km^alpha+max(min(Solubility/MW; DrugLiquid/Liquid);0)^alpha); P_int_trans*Aeff*P_int_trans_lum_cell_factor * min(Solubility/MW; DrugLiquid/Liquid)) + TM * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha+(K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='MucosaPgPToLumen_Hill';
-UPDATE tab_calculation_method_rates SET formula='Min(CP * f_cell * V * kcat * (K_water_int * max(C_int;0))^alpha / ( Km^alpha + (K_water_int * max(C_int;0))^alpha) ; PA_int_cell * K_water_int * C_int) + CP * f_cell * V * kcat * (K_water_cell * max(C_cell;0))^alpha / (Km^alpha + (K_water_cell * max(C_cell;0))^alpha)' WHERE calculation_method='LinksCommon' AND formula_rate='PgpSpecific_Hill';
+UPDATE tab_container_parameters SET sequence=2 WHERE container_id=5015 AND container_type='GENERAL' AND container_name='PROTEIN' AND parameter_name='Relative expression in plasma';

```

Comment:
Adding those guys here to ensure that we have the same display as in indivudal. This set to 1 for all of them thus the order was not always the same. Doof